webapp/app/.server/service/reports.ts

```diff
@@ -4,7 +4,9 @@ import { dirname } from "node:path";
 import { fileURLToPath } from "node:url";
```

Comment:
The only logic that I changed in the webapp (I think) is in this file. Basically now when a report is generated:

- First it checks there is a configuration to avoid generate reports (new env var). If that the case, an empty report is returned.
- Then tries to generate the report. If the report can be generated it returns the report.
- if something fails (any kind of exception) it logs the exception and returns and empty report without storing anything in the db.